spec.html

```diff
@@ -34811,15 +34811,15 @@ <h1>Properties of Set Iterator Instances</h1>
                 [[SetNextIndex]]
               </td>
               <td>
-                The integer index of the next Set data element to be examined by this iterator
+                The integer index of the next Set data element to be examined by this iterator.
               </td>
             </tr>
             <tr>
               <td>
                 [[SetIterationKind]]
               </td>
               <td>
-                A String value that identifies what is to be returned for each element of the iteration. The possible values are: `"key"`, `"value"`, `"key+value"`. `"key"` and `"value"` have the same meaning.
```

Comment:
`[[SetIterationKind]]` is never `"key"` because:

> The initial value of the `Set.prototype.keys` property is the same function object as the initial value of the `Set.prototype.values` property.